app/models/organization.rb

```diff
@@ -34,8 +35,16 @@ def self.estella_search_query
     OrganizationsQuery
   end
 
+  def name
+    names.first
+  end
+
+  def alternate_names
+    names.drop(1)
+  end
+
   def names
-    organization_names.pluck(:content)
+    organization_names.order(created_at: :desc).sort_by(&:rank).pluck(:content)
```

Comment:
The `order` doesn't do anything given a `sort_by` here, don't you want to order by both in SQL?

Reply:
I think all the other `_names` could benefit from a predictable sort as well, but that's beyond the scope of this PR.

Reply:
This looks correct to me - rank isn't stored in the database, it's computed on the fly and the created_at order breaks ties.

Reply:
Right, I would expect it to break ties and my spec confirms this - both the names there have the same rank.

Reply:
Keep in mind that Ruby's sort algorithm is not a stable sort (quick sort) and there could be an edge case that is extremely hard to find.

Reply:
My spec disagrees. This doesn't break any ties, it re-sorts a collection previously sorted by `-created_at` by `rank`. See https://github.com/artsy/bearden/pull/321.